data_ingestion/02_Ingest_data_with_Athena_v1.ipynb

```diff
@@ -0,0 +1,1144 @@
+{
```

Comment:
<p>Typo:</p><blockquote>making sure you catalog in the specified region contains the database. </blockquote><p>Needs "that"</p><blockquote>making sure you catalog in the specified region that contains the database. </blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1602/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>